hack/lib/Dockerfile-origin-release

```diff
@@ -3,7 +3,7 @@ FROM registry.svc.ci.openshift.org/openshift/origin-release:v4.0 as origin-relea
 FROM fedora:29 as jq
 RUN yum install -y jq
```

Comment:
this is pretty slow to me on rebuilds, I guess we want some custom `origin-build-tools` image that already have the  `jq` installed.